api/tests/unit/domain/models/Challenge_test.js

```diff
@@ -44,6 +44,8 @@ describe('Unit | Domain | Models | Challenge', function () {
         responsive: 'Smartphone',
         shuffled: false,
         alternativeVersion: 1,
+        accessibility1: 'OK',
+        accessibility2: 'KO',
```

Comment:
Préférer utiliser l'enum implémenté plus haut ?

```suggestion
       accessibility1: Accessibility.KO,
        accessibility2: Accessibility.OK,
```